pkg/extensions/search/resolver.go

```diff
@@ -722,40 +721,21 @@ func BuildImageInfo(repo string, tag string, manifestDigest godigest.Digest,
 	return imageInfo
 }
 
-// returns either a user has or not rights on 'repository'.
-func matchesRepo(globPatterns map[string]bool, repository string) bool {
```

Comment:
Why is this part removed?

Reply:
I moved it here: https://github.com/project-zot/zot/pull/951/files#diff-78f3d35f6fab9bf5af09ca967e0e55ec0ccab9cd7ab67fe5dfce6f51ad59f7a6R46
to be more explicit.